projects/plugins/jetpack/_inc/lib/class-jetpack-recommendations.php

```diff
@@ -349,7 +349,7 @@ public static function should_recommend_videopress( $site_plan, $site_products )
 		}
 
 		// Does this site already have a VideoPress product?
-		$has_videopress_product = count( array_intersect( array( 'jetpack_videopress', 'jetpack_videopress_monthly' ), $site_products ) ) > 0;
+		$has_videopress_product = array_intersect( array( 'jetpack_videopress', 'jetpack_videopress_monthly' ), $site_products ) !== array();
```

Comment:
Adding a check to this one to return out early if `$site_products` ends up null.